### PR TITLE
UI: replace npm-run-all with concurrently

### PR DESCRIPTION
**Pull request by hashishaw**
Replace `npm-run-all` with `concurrently`

example of `yarn fmt` output:
Pretty colors

For the test jobs (non-quick) a lint error will cancel the rest of the tests:
Example of lint failure canceling the test run

### CODE REVIEW

File: ui/yarn.lock [hunk 1]
```diff
@@ -25649,13 +25555,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"shell-quote@npm:^1.6.1":
```
Comment (Monkeychip) [inline ui/yarn.lock:25652]: 🥳

**Review by Monkeychip — APPROVED** [review]
✅ Assuming you confirmed the scripts still work as expected. Thanks for tackling this one so quickly!

**Comment by hashishaw** [comment]
> ✅ Assuming you confirmed the scripts still work as expected. Thanks for tackling this one so quickly!

Yes, I tested locally but wanted to run it through CI to make sure that all works as expected. I'll post screenshots of the output soon, it's pretty!

File: ui/yarn.lock [hunk 2]
```diff
@@ -13251,7 +13271,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"date-fns@npm:^2.29.2":
+"date-fns@npm:^2.29.2, date-fns@npm:^2.30.0":
```
Comment (hashishaw) [inline ui/yarn.lock:13274]: Todo: Look at consolidating versions

File: ui/yarn.lock [hunk 3]
```diff
@@ -28924,7 +28828,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"yargs@npm:^17.7.1":
+"yargs@npm:^17.7.1, yargs@npm:^17.7.2":
```
[the remark this answers is not in the document]
Comment (hashishaw) [reply ui/yarn.lock:28831]: ember-template-lint is on of the ones setting the version to 1.17.1 -- will probably be updated with forthcoming ember upgrades

File: ui/yarn.lock (hunk 2 above)
Comment (hashishaw) [reply ui/yarn.lock:13267]: ember-template-lint is also the culprit here

**Pull request merged**